docs/articles/deployments.md

```diff
@@ -0,0 +1,237 @@
+                ]
+            }
+        ],
+        "currentActions": [
+            {
+                "action": "ScaleApplication",
+                "app": "/product/service/rails-app"
+            }
+        ],
+        "currentStep": 2,
+        "totalSteps": 2
+    }
+]
+```
+This endpoint returns a list of currently active deployments (some fields omitted for brevity). Interesting fields are:
+- `id` - this is our `deploymentId` from the original response
+- `affectedApps` - a deployment may actually "affect" (add, remove or modify) multiple service definitions (e.g. in a case of a group deployment)
+- `steps` - a series of actions to achieve the defined goal. This is being referred to as a [deployment plan](deployments.md#deployment-plan)
+- `currentAction` - currently executed action of the plan
+
+## Deployment plan
+In the above example `steps` field holds the deployment plan. Our deployment plan has two deployment steps: `StartApplication` and `ScaleApplication`. Currently, following deployment steps exist:
+
+- `StartApplication` - this step exists for backward compatibility reasons. Currently, it is doing nothing and is immediately successful
+- `ScaleApplication` - is responsible for starting the defined number of tasks and waiting for them to be up and running
+- `StopApplication` - stops given service by killing its tasks and waiting for the kill confirmation
+- `RestartApplication` - used when modifying an existing service definition and restarting its tasks (using the new definition)
+
+Steps are executed sequentially, however actions within one step (should there be more than one) are executed in parallel. Once all steps are executed successfully, the deployment is finished and removed from the state.
+
+## Canceling a deployment
+Sometimes a deployment is "stuck" for some reason (see [troubleshooting](troubleshooting.md) for more information). In this case, it might be helpful to cancel the deployment, returning to the previous state. Let's cancel the above stuck deployment by sending a `DELETE /v2/deployments/3bcb15fc-737f-4a36-b185-2ae3a703239e` request. The response is `HTTP 200(ok)` but the payload might be somewhat a surprise:
+```
+{
+    "version": "2018-12-05T00:28:00.584Z",
+    "deploymentId": "342a6dc5-f871-4d10-a4ec-bbe21b431d58"
+}
+```
+We see that another `deploymentId` is returned. This is because internally canceling a deployment simply creats another deployment that reverts the steps of the original one.
+
+**Known caveats:** Marathon has a cap on the number of concurrently running deployments. This is controlled by `--max_running_deployments` [command line argument](command-line-arguments.md). In some rare occasions, one can have the maximum number of running deployments achieved so that canceling one is not possible (since a new deployment has to be created). In this case, canceling using `?force=true` parameter aka. [forced deployment](deployments.md#forced-deployments) should be used.
```

Comment:
```suggestion
**Known caveats:** Marathon has a cap on the number of concurrently running deployments. This is controlled by `--max_running_deployments` [command line argument](command-line-arguments.md). In some rare cases it's possible to reach this limit so that canceling a deployments is not possible (since a new deployment has to be created). In this happens, canceling using `?force=true` parameter aka. [forced deployment](deployments.md#forced-deployments) should be used.
```